frontend/src/components/molecules/ColumnMenuFilter/index.jsx

```diff
@@ -14,11 +14,16 @@ import { uid } from "@/utils/helper";
 import { useSelector } from "react-redux";
 
 /**
+ * A component that displays a list of columns available for the project
+ * ID type, and allows the user to select or deselect them.
+ * Selection changes are communicated through the callback.
+ * 
  * @param {*} props
- *  - type: The project ID type within the CPT Dashboard.
- *  - setColumns: A callback with two inputs, value and isAdding:
+ *  - type: The project ID type within the CPT Dashboard that as used in
```

Comment:
"that *is* used"? Or should the sentence be structured differently?

Reply:
That looks like it ended up as a mix of "is is used in" and "as used in". I switched it to "as used in".